lib/Local.js

```diff
@@ -179,30 +184,27 @@ function Local(){
         break;
 
       case 'forceproxy':
+      case 'forceProxy':
         if(value)
-          this.forceProxyFlag = '-forceproxy';
-        break;
-
-      case 'hosts':
-        if(value)
-          this.hosts = value;
+          this.forceProxyFlag = '--force-proxy';
         break;
 
       case 'logfile':
+      case 'logFile':
         if(value)
           this.logfile = value;
         break;
 
-      case 'binarypath':
```

Comment:
Any reason to remove this option? This option allows us to customize the BrowserStackLocal binary path ( the binary must be present at the path and won't be downloaded ).

We were using this for other language bindings as well.